lib/populate/extract-ban.js

```diff
@@ -81,16 +21,9 @@ async function extractFromApiDepot(codeCommune) {
 }
 
 async function extract(codeCommune) {
-  const data = await extractFromApiDepot(codeCommune)
+  const data = await extractFromApiDepot(codeCommune) || await extractFromBAN(codeCommune)
 
   if (data) {
```

Comment:
Attention il faut inverser la condition ici
```suggestion
  if (!data) {
```

Reply:
Il faut aussi retourner `data`.